src/app/components/Text/index.test.jsx

```diff
@@ -35,9 +37,11 @@ describe('Text', () => {
 
   shouldMatchSnapshot(
     'should render a <figcaption> as per the Caption component override',
-    <Text
-      text="This is text that contains an [inline link](https://www.bbc.com/news) inside it."
-      paragraphOverride={Caption}
-    />,
+    <ServiceContext.Provider value={serviceContextStub.news}>
```

Comment:
Yes, exactly that. We need to pass the custom value to `ServiceContext` for the tests, but having the simpler interface of `ServiceContextProvider` and `ServiceContextConsumer` inside the components makes sure we don't pass in the services object into the components.

Reply:
Okay, as this is only used for testing, it might be cleaner and simpler to write a Jest mock for the config files used by the `ServiceContext`.

This would mean we don't need to export the context just for testing purposes, which we shouldn't do really, and there can be a consistent usage of the service context (across tests and app code), and a consistent way to test.